### ISSUE TITLE
Top Panel accessibility issue: `First Image` and `Previous Image` buttons

### ISSUE THREAD

**Issue by LanieOkorodudu** [UniversalViewer/universalviewer#1536]
**UV version:**
 `universalviewer@dev`

The problems I previously mentioned in #1496 with the top panel navigation remain unresolved in the latest version. Both screen reader support and keyboard navigation are affected.

**I'm submitting a:**

- [x] bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below
- [ ] feature request => please use the [user stories repo](https://github.com/UniversalViewer/user-stories)
- [ ] support request => Please do not submit support requests here, use [stackoverflow](http://stackoverflow.com/questions/tagged/iiif)

**Current behavior:**
- `v4.0.0`: The `First Image` and `Previous Image` buttons were detected and announced correctly.
- `v4.2.1`: These buttons are skipped entirely, they are not reachable via keyboard navigation and are not announced by the screen reader.

**Expected behavior:**
- The `First Image` and `Previous Image` buttons should be included in the tab order.
- Screen readers should detect and announce these buttons as interactive elements.

**Steps to reproduce:**
1. Navigate to the top panel using either a screen reader or keyboard navigation.
2. Attempt to access the First Image and Previous Image buttons.

**Related code:**

```
// insert any relevant code here
```

**Other information:**
I will attach a screenshot showing the keyboard navigation for reference.
`v4.0.0` screenshot for keyboard navigation
<img width="1210" height="873" alt="Image" src="https://github.com/user-attachments/assets/5837d073-37ae-4795-a769-da732f9d5ac9" />

When pressing the Tab key, focus first highlights the `Skip to download and alternative formats` option. Pressing Tab again, I would expect the next item in the tab order to be the `First Image` button. Instead, focus jumps to the `Search by Page Number` field, as shown in the screenshot below.
`v4.2.1`screenshot for keyboard navigation
<img width="1232" height="981" alt="Image" src="https://github.com/user-attachments/assets/a2a2e037-fe60-431d-a2eb-bd771211f090" />

**Comment by LanieOkorodudu** [comment]
Upon investigating the issue and comparing v4.0.0 with v4.2.1:
- In `v4.0.0`, the “**First Image**” and “**Previous Image**” buttons in the top panel are reported as **focusable: true,** so they are included in the tab order.
- In `v4.2.1`, the same buttons are rendered with **disabled: true,** which makes them non-focusable and removes them from the tab order.
As a result, when pressing the **Tab** key in `v4.2.1`, focus skips over the “**First Image**” and “**Previous Image**” buttons. In `v4.0.0`, these buttons were still reachable with Tab navigation.

Here is the screenshot comparing the Accessibility Tree view in the DOM for both versions:
(attach screenshot `v4.0.0 here)
<img width="332" height="120" alt="Image" src="https://github.com/user-attachments/assets/e7a05a60-67df-4a23-9d4f-8cadc2af015c" />

(attach screenshot `v4.21`0 here)
<img width="284" height="111" alt="Image" src="https://github.com/user-attachments/assets/db56dd15-fe8c-425c-ace7-cbe5a3ecef97" />

**Comment by demiankatz** [comment]
@LanieOkorodudu, the difference you describe actually appears to be a bug in 4.0. The "first" and "previous" buttons should be disabled while viewing page 1 of an object, because there is no previous page to navigate to. We don't want to tab to a button that does not do anything. You should be able to tab to the "next" and "last" buttons -- and once you have moved past page 1, the "previous" and "first" buttons should be enabled.

Are you seeing something different? The current code in https://universalviewer.dev seems to be behaving appropriately for me in Firefox, at least.

**Comment by LanieOkorodudu** [comment]
If the behavior in v4.0.0 (where the “First” and “Previous” buttons were still focusable while viewing page 1) is considered a bug, then the issue I raised is not actually relevant. In v4.0.0, when I pressed Tab on page 1, it would still announce “First,” then “Previous,” and then “Next” — even though “First” and “Previous” had no action at that state.

Since the intended behavior is that _we don’t want to tab to a button that does not do anything_, the current behavior in v4.2.1 (skipping “First” and “Previous” while on page 1) is correct.

I also checked the dev version, and I see the same as you described: once I move past page 1, the “Previous” and “First” buttons become enabled and focusable.

So this can be considered resolved, and no further action is needed on this issue I believe?

**Comment by demiankatz** [comment]
I believe so, unless somebody with greater accessibility expertise than myself believes that there is an advantage to making the disabled buttons more available to keyboard interactions.

**Comment by erinburnand** [comment]
Agree that this issue can be closed

**Comment by demiankatz** [comment]
Okay, I'll close for now. If anyone has remaining concerns, we can always reopen in future.
